website/docs/r/vpc_group.html.markdown

```diff
@@ -0,0 +1,5 @@
+---
+subcategory: "VPC"
+layout: "nsxt"
+page_title: "NSXT: nsxt_vpc_group"
+description: A resource to configure a VPC Group and its members.
```

Comment:
How should it work here? Do we need to link to the policy_group resource page?

Reply:
That requires completion